benchmarks/src/main/scala/zio/stm/TMapContentionBenchmarks.scala

```diff
@@ -1,8 +1,7 @@
 package zio.stm
 
 import org.openjdk.jmh.annotations._
-import zio._
-import zio.clock.Clock
+import zio.{Clock, _}
```

Comment:
```suggestion
import zio._
```

Reply:
Ah, I knew a few of these must've escaped my (and scala-fix's) wrath.